depends/packages/qt.mk

```diff
@@ -252,23 +247,26 @@ define $(package)_config_cmds
   ./configure $($(package)_config_opts) && \
   cd .. && \
   $(MAKE) -C qtbase sub-src-clean && \
-  qtbase/bin/qmake -o qttranslations/Makefile qttranslations/qttranslations.pro && \
-  qtbase/bin/qmake -o qttranslations/translations/Makefile qttranslations/translations/translations.pro && \
   qtbase/bin/qmake -o qttools/src/linguist/lrelease/Makefile qttools/src/linguist/lrelease/lrelease.pro && \
   qtbase/bin/qmake -o qttools/src/linguist/lupdate/Makefile qttools/src/linguist/lupdate/lupdate.pro
 endef
 
 define $(package)_build_cmds
   $(MAKE) -C qtbase/src $(addprefix sub-,$($(package)_qt_libs)) && \
   $(MAKE) -C qttools/src/linguist/lrelease && \
-  $(MAKE) -C qttools/src/linguist/lupdate && \
-  $(MAKE) -C qttranslations
+  $(MAKE) -C qttools/src/linguist/lupdate
 endef
 
+# Qt translations module depends on lrelease binary being installed,
+# therefore all translations processing is put here.
```

Comment:
I think this comment needs to either be more explanatory, or just removed.

Reply:
I mean [that](https://github.com/bitcoin/bitcoin/pull/21420#discussion_r596444552).

Remove it?